Apps/W1/Shopify/app/src/Companies/Enums/ShpfyCompanyMapping.Enum.al

```diff
@@ -13,6 +13,11 @@ enum 30151 "Shpfy Company Mapping" implements "Shpfy ICompany Mapping"
         Caption = 'By Email/Phone';
         Implementation = "Shpfy ICompany Mapping" = "Shpfy Comp. By Email/Phone";
     }
+    value(1; "By Tax Id")
```

Comment:
I'd be wary of reusing a gap in an enum value. Maybe it used to be here, and someone might still have a value of 1 in their DB?
Maybe not, and it's an unlikely edge case. But just a thought.

Reply:
I don't really like to have gaps, but like you said 'maybe there is a reason' and maybe enum is extended somewhere with value 1.
let's moved it to '3'.